barretenberg/cpp/src/barretenberg/benchmark/relations_bench/relations.bench.cpp

```diff
@@ -1,7 +1,7 @@
-#include "barretenberg/flavor/ecc_vm.hpp"
-#include "barretenberg/flavor/goblin_translator.hpp"
-#include "barretenberg/flavor/goblin_ultra.hpp"
-#include "barretenberg/flavor/ultra.hpp"
+#include "barretenberg/eccvm/eccvm_flavor.hpp"
+#include "barretenberg/stdlib_circuit_builders/goblin_ultra_flavor.hpp"
```

Comment:
I dont quite get why the flavor should be in the stdlib_circuit_builders since the circuit builders are not templated by flavor but maybe I am missing sth

Reply:
Flavor and circuit builder go side-by-side now--they have to be designed together, anyway.